pages/desktop/regions/debug.py

```diff
@@ -18,4 +19,6 @@ def __init__(self, testsetup):
         self.wait_for_element_visible(*self._region_select_locator)
 
     def select_region(self, region):
-        self.select_option(region, self._region_select_locator)
+        Select(
+            self.selenium.find_element(
+                *self._region_select_locator)).select_by_value(region)
```

Comment:
I'm not keen on the style here, but r+ if this is working (I haven't run the tests, let me know if you would like me to - or perhaps trigger an adhoc job on staging)